### PR TITLE
[add] Update Incomplete Events

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?
If chaos is deleted before an event is over, then the incomplete event would be stored in dbtastore, which means the event would never save the finish_time.
UpdateIncompleteEvents can update the finish_time when the chaos is deleted.

### What is changed and how does it work?
add UpdateIncompleteEvents

### Check List

Tests

- No code

Code changes

- Has Go code change

Side effects

- NO

Related changes

- NO

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/store/event/event.go [hunk 1]
```diff
@@ -423,6 +423,16 @@ func (e *eventStore) getUID(_ context.Context, ns, name string) (string, error)
 	return UID, nil
 }
 
+// UpdateIncompleteEvents updates the incomplete event by the namespace and name
+func (e *eventStore) UpdateIncompleteEvents(_ context.Context, ns, name string) error {
+	return e.db.Model(core.Event{}).
+		Where(
+			"namespace = ? and experiment = ? and finish_time IS NULL",
```
Comment (cwen0) [inline pkg/store/event/event.go:430]: Do we need to handle the `Not found` error?

Comment (fewdan) [reply pkg/store/event/event.go:430]: I just tried it and there is no "not found error". I don't know if it is because update has no such errors.

Comment (cwen0) [reply pkg/store/event/event.go:430]: Got

**Pull request merged**